packages/visx-brush/src/BaseBrush.tsx

```diff
@@ -111,6 +125,148 @@ export default class BaseBrush extends React.Component<BaseBrushProps, BaseBrush
     }
   }
 
+  componentDidMount() {
+    if (this.props.useWindowMoveEvents) {
+      window.addEventListener('mouseup', this.handleMouseUp);
+      window.addEventListener('mousemove', this.handleMouseMove);
+    }
+  }
+
+  componentWillUnmount() {
+    if (this.props.useWindowMoveEvents) {
+      window.removeEventListener('mouseup', this.handleMouseUp);
+      window.removeEventListener('mousemove', this.handleMouseMove);
+    }
+  }
+
+  handleMouseUp = () => {
```

Comment:
could we make it clear that these are window handlers? (and could generalize to pointer events)

```suggestion
  handleWindowPointerUp = () => {
```